cont3xt/vueapp/src/components/links/LinkGroupCard.vue

```diff
@@ -420,6 +420,25 @@ export default {
       return normalizedLinkGroup;
     },
     getUrl (url) {
+      const types = ['domain', 'ip', 'url', 'email', 'hash', 'phone', 'text'];
+      // replaces ${arrayType} with a comma-separated list of all indicators of that type
+      // for example ${arrayIp} gets replaced with all the IPs in the results
+      for (const type of types) {
+        const matchStr = `\\$\\{array${type.charAt(0).toUpperCase() + type.slice(1)}\\}`;
```

Comment:
you didn't ask, but why not just make the array ['arrayDomain', ...] :)

Reply:
because I use the type twice in the loop. to make the regex and then to determine if it's the correct type to add to the array